code/__DEFINES/_macros.dm

```diff
@@ -125,13 +125,18 @@
 /// General I/O helpers
 #define to_target(target, payload)                          target << (payload)
 #define from_target(target, receiver)                       target >> (receiver)
-#define to_file(file_entry, file_content)                   file_entry << file_content
+#define to_file(file_entry, file_content)
```

Comment:
Why is this macro defined to null?

Reply:
I think I accidentally deleted it. I'll put it back.

Reply:
I hope I've gotten things back on track.